rooms.yml

```diff
@@ -158,6 +158,7 @@ stackexchange.com:
       - 17078244 # Daniil Manokhin
       - 8629256  # M--
       - 15062967 # Adrian Mole
+      - 195522  #luuklag
```

Comment:
The number here needs to be the user's global account ID, which is the first column in users.yml. In this case, that number is 1599858.